### ISSUE TITLE
Backend code

### ISSUE THREAD

**Issue by seniorOtaka** [UIT-Resources/Xamarin-Ecommerce-App#2]
Hello
I like your effort in this app and i wonder if you can share the backend code

**Comment by UIT-Resources** [comment]
Definitely, Yes. I'm glad you care about it. But I'm still confused about your idea. You want to have backend code of the server-side. is it right?

**Comment by seniorOtaka** [comment]
yes, and i will be very thankful for that

**Comment by UIT-Resources** [comment]
Hi, guy. 

we're sorry when we have to tell you that we can't share that code because, at the present, our server has been edit to serve a different purpose and unfortunately, we don't have any backup of this version. I think you should build a similar server and connect to this app, It will work fluently. 

Sincerely!!

**Comment by seniorOtaka** [comment]
Oh, my bad luck
no problem and thanks for your effort
i will work on the server version and will share with you to include it in your repo if you still interested to make the project open sourced

**Comment by UIT-Resources** [comment]
Of course, yes and I want to tell you that this version of the app is not really completed when compare with my expected design and also has some performance issue, so I really want to continue to improve and develop this app. But, we don't have much time at present. So I hope that you will improve this app. I will so glad if I see your better version of this app in the future.
Thank you for your attention.
